README.md

```diff
@@ -1,2 +1,216 @@
-# Rails-Capstone-Project---Budget-App
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Rails-Capstone-Project---Budget-App <a name="about-project"></a>
+
 The Ruby on Rails capstone project, the objective is to create a mobile web application that enables the user to effectively manage their budget. The application will allow the user to track their transactions and categorize them, providing insights into their spending habits and helping them to understand where their money is being spent.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/ruby/erb">Ruby ERB</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://tailwindcss.com/">Tailwindcss</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby On Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **User-friendly interface**
+- **Add new expenses and categorize them according to your needs**
+- **All data are private to the logged-in user**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-app-5iw8.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- For the deploy issue, I'll advise you try hosting website. You can try [Railway](https://docs.railway.app/getting-started) I think this will help. 😉

Reply:
@rotimiazeez I am getting the same error can you help me out to deploy the project